cmd/run.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/ashigirl96/jscript/pkg"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "A brief description of your command",
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		manager, err := pkg.GetPackageManager()
+		if err != nil {
+			return err
+		}
+		output, err := manager.Run(args[0])
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		fmt.Println(output)
```

Comment:
[]byte で返ってくるので、 stringにキャスト必要かも？
```suggestion
		fmt.Println(string(output))
```